packages/form-js-viewer/src/render/components/FormField.js

```diff
@@ -6,41 +6,47 @@ import {
 import {
   FormContext,
   FormRenderContext
-} from './context';
+} from '../context';
 
 import {
   findData,
   findErrors,
   pathStringify
-} from '../util';
+} from '../../util';
 
 const noop = () => false;
 
 
-export default function FormElement(props) {
+export default function FormField(props) {
   const {
     path,
     field,
     onChange
   } = props;
 
+  const {
+    getService
+  } = useContext(FormContext);
+
+  const formFieldRegistry = getService('formFieldRegistry'),
```

Comment:
Why not `useService` here?

Reply:
Done (cf. https://github.com/bpmn-io/form-js/pull/75/commits/ef32a209c86feff9b4166cd79e9bd1c6a31d219a).